solr/core/src/test/org/apache/solr/handler/ReplicationTestHelper.java

```diff
@@ -111,7 +108,7 @@ public static void assertVersions(SolrClient client1, SolrClient client2) throws
     Long maxVersionClient2 = getVersion(client2);
 
     if (maxVersionClient1 > 0 && maxVersionClient2 > 0) {
-      assertEquals(maxVersionClient1, maxVersionClient2);
+      SolrTestCaseJ4.assertEquals(maxVersionClient1, maxVersionClient2);
```

Comment:
Why not just extend `SolrTestCase` like the other files?

Reply:
its a test helper. no real tests so seemed weird

Reply:
This also goes for a few others (I think ~5 classes) that are test utils only and seemed weird to make extend `SolrTestCase`.